runtime-config-linux.md

```diff
@@ -131,6 +131,60 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+The path to the cgroups can to be specified in the Spec via `cgroupsPath`.
+`cgroupsPath` is expected to be relative to the cgroups mount point.
+If not specified, cgroups will be created under '/'.
+Implementations of the Spec can choose to name cgroups in any manner.
```

Comment:
I feel it's clear enough for now.

On Thu, Sep 10, 2015 at 2:55 PM, W. Trevor King <notifications@github.com>
wrote:

> In runtime-config-linux.md
> <https://github.com/opencontainers/specs/pull/137#discussion_r39198980>:
>
> > @@ -131,6 +131,60 @@ Also known as cgroups, they are used to restrict resource usage for a container
> >  cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
> >  For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
> >
> > +The path to the cgroups can to be specified in the Spec via `cgroupsPath`.
> > +`cgroupsPath` is expected to be relative to the cgroups mount point.
> > +If not specified, cgroups will be created under '/'.
> > +Implementations of the Spec can choose to name cgroups in any manner.
>
> This is only true if cgroupsPath is unset. So maybe rephrase as “In that
> case, implementations can choose to name cgroups in any manner.” to tie it
> into the previous ”If not specified…” sentence.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/137/files#r39198980>.
>